packages/beacon-node/src/monitoring/system.ts

```diff
@@ -1,5 +1,9 @@
+// We want to keep `system` export to be used as namespace
+/* eslint-disable import/no-named-as-default-member */
 import os from "node:os";
 import path from "node:path";
+// We want to keep `system` export as it's more readable and easier to understand
+// eslint-disable-next-line import/no-named-as-default
```

Comment:
this seems redundant now, and should probably add the comment on top of the file here, next to the import itself

Reply:
It's not redundant, those are two different rules `import/no-named-as-default-member` and `import/no-named-as-default`

Reply:
ah right those are differnet